magma/lib/magma/censor.rb

```diff
@@ -23,6 +23,19 @@ def censored_reasons(model, record_set)
         restrict: true
       ).answer.map(&:last)
 
+      existing_identifiers = Magma::Question.new(
```

Comment:
This fixes a bug I discovered where a non priveledged user could not create new magma records since it would trigger the restricted check in the censor.  Matching spec found.

The idea is to identify restricted records as records that both don't show up from a restricted query AND already exist in the database.

Reply:
Nice find!

Reply:
Hmm, perhaps a bad result of having most of the update_spec tests run with the permissions `editor: { perm: 'E:labors' }`. There appear to be serious problems here, even with @corps's fix, there are many failing tests (besides the censor ones) if you change the above perm to `e:labors`. I'm investigating...